Makefile

```diff
@@ -352,6 +352,8 @@ clean-proto-generated-srcs:
 config-controller-gen:
 	make -C config-controller/ manifests
 	make -C config-controller/ generate
+	echo '{{- include "srox.init" . -}}' > image/templates/helm/stackrox-central/templates/00-securitypolicies-crd.yaml
```

Comment:
We could try this: 

```
diff --git a/config-controller/Makefile b/config-controller/Makefile
index 0934bac044..99d9817599 100644
--- a/config-controller/Makefile
+++ b/config-controller/Makefile
@@ -43,9 +43,11 @@ help: ## Display this help.

 ##@ Development

+HELMDIR = ../image/templates/helm/stackrox-central/templates
 .PHONY: manifests
 manifests: controller-gen ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
-       $(CONTROLLER_GEN) rbac:roleName=manager-role crd webhook paths="./..." output:crd:artifacts:config=config/crd/bases
+       $(CONTROLLER_GEN) rbac:roleName=manager-role crd:headerFile=crdHeader webhook paths="./..." output:crd:artifacts:config=$(HELMDIR)
+       mv $(HELMDIR)/config.stackrox.io_securitypolicies.yaml $(HELMDIR)/00-securitypolicies-crd.yaml
```

This would also break `make install` and `make build-installer`, though I'm not sure how important those make targets are to us.

Reply:
Imho this is fine but an additional line to mention where is filed was generated would be great.

For example: 

```suggestion
	echo '# Generated by make config-controller-gen\n{{- include "srox.init" . -}}' > image/templates/helm/stackrox-central/templates/00-securitypolicies-crd.yaml
```